spec/unit/recipes/configure_spec.rb

```diff
@@ -258,9 +259,7 @@
     end
 
     it 'creates proper database.yml template' do
-      db_config = Drivers::Db::Sqlite.new(
-        aws_opsworks_app(data_sources: []), dummy_node, rds: aws_opsworks_rds_db_instance(engine: 'sqlite3')
-      ).out
+      db_config = Drivers::Db::Sqlite.new(aws_opsworks_app(data_sources: []), dummy_node).out
```

Comment:
It was a bit backward to fake an RDS to implement a Sqlite database, but understandable as the chef libraries were designed to only work with RDS.